### PR TITLE
[docs] Add a TPU example to the docs

### PR DESCRIPTION
**Pull request by Yard1**
## Why are these changes needed?

Adds the example TPU config to the docs.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/cluster/config.rst [hunk 1]
```diff
@@ -1163,3 +1163,15 @@ Full configuration
 
         .. literalinclude:: ../../../python/ray/autoscaler/gcp/example-full.yaml
             :language: yaml
+
+TPU Configuration
+~~~~~~~~~~~~~~~~~
+
+It is possible to use `TPU VMs <https://cloud.google.com/tpu/docs/users-guide-tpu-vm>`_ on GCP. Currently, TPU pods are not supported.
```
Comment (ijrsvt) [inline doc/source/cluster/config.rst:1170]: Could you just add a link to `TPU pods` documentation?

**Review by ijrsvt — COMMENTED** [review]
Is there any additional GCP setup needed for this?

**Comment by Yard1** [comment]
It requires the TPU API to be enabled - I can mention that in the doc.

**Comment by ijrsvt** [comment]
@richardliaw Is this the correct location for this?

**Comment by Yard1** [comment]
I'm open to feedback on the location - I couldn't find a good place for this doc, and this file seemed to be the least bad option.

**Review by ijrsvt — APPROVED** [review]
This looks good to me!

**Pull request merged**